docs/content/filescom.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Files.com"
+description: "Rclone docs for Files.com"
+versionIntroduced: "v1.68"
+---
+
+# {{< icon "fa fa-file-alt" >}} Files.com
+
+[Files.com](https://www.files.com/) is a cloud storage service that provides a secure and easy way to store and share files.
+
+Paths are specified as `remote:path`
+
+Paths may be as deep as required, e.g. `remote:directory/subdirectory`.
```

Comment:
Can you move these lines to the how to use section below?

They've been copied around from backend docs and I'm trying to get rid of them!

Reply:
I should probably change the box backend.

I suggest you just remove those lines to make the docs read better.

Reply:
Updated.